azure/packages/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -6,6 +6,8 @@ import { strict as assert } from "assert";
 import { AttachState } from "@fluidframework/container-definitions";
 import { ContainerSchema } from "@fluidframework/fluid-static";
 import { ISharedMap, IValueChanged, SharedMap } from "@fluidframework/map";
+import { ConnectionState } from "@fluidframework/container-loader";
```

Comment:
Can't you pull this from fluid-framework now since #10686 was merged?

Reply:
We could, but we already have container-loader in package.json and have not imported fluid-framework, so I feel like it makes sense to leave it here.

Reply:
Should we create a work item to track updating the imports separately?